exercises/sgf-parsing/canonical-data.json

```
@@ -170,6 +170,185 @@
+        {
+            "uuid": "deaecb9d-b6df-4658-aa92-dcd70f4d472a",
+            "description": "within property values, newlines remain as newlines",
+            "property": "parse",
+            "input": {
+                "encoded": "(;A[hello\n\nworld])"
+            },
+            "expected": {
+                "properties": {
+                    "A": ["hello\n\nworld"]
+                },
+                "children": []
+            }
+        },
+        {
+            "uuid": "8e4c970e-42d7-440e-bfef-5d7a296868ef",
+            "description": "Escaped closing bracket within property value becomes just a closing bracket",
+            "property": "parse",
+            "input": {
+                "encoded": "(;A[\\]])"
+            },
+            "expected": {
+                "properties": {
+                    "A": ["]"]
+                },
+                "children": []
+            }
+        },
+        {
+            "uuid": "cf371fa8-ba4a-45ec-82fb-38668edcb15f",
+            "description": "Escaped backslash in property value becomes just a backslash",
+            "property": "parse",
+            "input": {
+                "encoded": "(;A[\\\\])"
+            },
+            "expected": {
+                "properties": {
+                    "A": ["\\"]
+                },
+                "children": []
+            }
+        },
+        {
+            "uuid": "dc13ca67-fac0-4b65-b3fe-c584d6a2c523",
+            "description": "Opening bracket within property value doesn't need to be escaped",
+            "property": "parse",
+            "input": {
+                "encoded": "(;A[x[y\\]z][foo]B[bar];C[baz])"
+            },
+            "expected": {
+                "properties": {
+                    "A": ["x[y]z", "foo"],
+                    "B": ["bar"]
+                },
+                "children": [
+                    {
+                        "properties": {
+                            "C": ["baz"]
+                        },
+                        "children": []
+                    }
+                ]
+            }
+        },
+        {
+            "uuid": "a780b97e-8dbb-474e-8f7e-4031902190e8",
+            "description": "Semicolon in property value doesn't need to be escaped",
+            "property": "parse",
+            "input": {
+                "encoded": "(;A[a;b][foo]B[bar];C[baz])"
+            },
+            "expected": {
+                "properties": {
+                    "A": ["a;b", "foo"],
+                    "B": ["bar"]
+                },
+                "children": [
+                    {
+                        "properties": {
+                            "C": ["baz"]
+                        },
+                        "children": []
+                    }
+                ]
+            }
+        },
+        {
+            "uuid": "0b57a79e-8d89-49e5-82b6-2eaaa6b88ed7",
+            "description": "Parentheses in property value don't need to be escaped",
```

For consistency with the other test cases:

```suggestion
            "description": "parentheses in property value don't need to be escaped",
```